BUILD.md

```diff
@@ -233,7 +233,7 @@ command:
 
 ```bash
 sudo dnf install git cmake libxcb-devel libX11-devel xcb-util-keysyms-devel \
-        libXrandr-devel wayland-devel zlib-devel lz4-devel libzstd-devel
+        libXrandr-devel wayland-devel zlib-devel lz4-devel libzstd-devel clang
```

Comment:
This looks like the project is choosing clang over gcc explicitly. Is there no way to ask Fedora to install whatever the system's default compiler is as we do for Debian-derived distributions with the `build-essential` package?

Reply:
https://fedoraproject.org/wiki/Releases/36/ChangeSet implies `gcc` is the default compiler.  https://unix.stackexchange.com/a/1344  says the command to install the development system is `sudo dnf groupinstall "Development Tools" "Development Libraries"`.  Although the source should compile under `clang` just fine, if there's a generic "install the compiler" target, we should require that, but if there isn't one, it looks like `gcc` is the compiler we should add here.

Reply:
https://fedoraproject.org/wiki/Changes/CompilerPolicy#:~:text=The%20default%20compiler%20for%20Fedora,the%20default%20compiler%20is%20clang appears to have a stronger prescription that `gcc` is the default.